client/utils.c

```diff
@@ -759,3 +759,71 @@ TDNFGetCmdOpt(
     return dwError;
 }
 
+GSList* TDNFMergeList(GSList* listA, GSList* listB)
```

Comment:
Is there no glib function that does this already?

Reply:
They have for structure not for list.